### ISSUE TITLE
Access Violation Error

### ISSUE THREAD

**Issue by zagnut84** [cheat-engine/cheat-engine#2488]
![Cheat Error](https://user-images.githubusercontent.com/125593199/219440481-2e199a7a-85f1-46dc-877b-af74de08ce5a.jpg)

I'm getting an access violation error. If I click OK. I get a BSOD.

**Comment by cheat-engine** [comment]
run the reset settings exe before you start CE

does it still bsod?   Then it's your anti virus or anti cheat

**Comment by zagnut84** [comment]
Done. Same error. I've got SAV app running but turned off AV services along with other feature sets. Should I shut down the app completely?

**Comment by Baka-Neko-Hime** [comment]
Hello, I got the same problem with 7.5 & IDK why whaterver on Win 7 or 10 or 11.
Do that need install externe tools like Net Framework or Visual C++ or whatever ?
But with 7.4, got no problem.

**Comment by LoGicScr** [comment]
Okay ,i'm experimenting the same error, i don't know what's the cause of this error but i'll continue find solution.

**Comment by LoGicScr** [comment]
> Hello, I got the same problem with 7.5 & IDK why whaterver on Win 7 or 10 or 11. Do that need install externe tools like Net Framework or Visual C++ or whatever ? But with 7.4, got no problem.

Nah, it's just probably error of this version, i've .NET Framework updated and visual c++ too and the error it's still.

**Comment by cheat-engine** [comment]
What is the BSOD errorcode ?

**Comment by zagnut84** [comment]
Hi again.

Sorry, I was not able to capture it. Happened just the one time. Once I changed the help file names the issue was resolved.

TY again for the help!
________________________________

**Comment by maingocvinh** [comment]
I'm using old cpu (Pentium E2200) and some feature does not support like Instruction set extensions: SSE, MMX but work on new CPU. Try use old Cheat Engine version 7.4
